src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -142,7 +142,7 @@ def skimToDataTier(cmsswVersion, skim):
         import subprocess
 
         p = subprocess.Popen("/cvmfs/cms.cern.ch/common/cmsos", stdout=subprocess.PIPE, shell=True)
-        cmsos = p.communicate()[0].strip()
+        cmsos = p.communicate()[0].decode('utf-8').strip()
```

Comment:
I'd suggest to use the str/bytes utils that we have been adopting in the code with the python migration, so something like:
```suggestion
        cmsos = p.communicate()[0]
        cmsos = decodeBytesToUnicodeConditional(cmsos, condition=PY3).strip()
```

Keep in mind you need to import them as well:
```
from Utils.PythonVersion import PY3
from Utils.Utilities import decodeBytesToUnicodeConditional
```

Reply:
Done